src/core/index.js

```diff
@@ -275,6 +266,40 @@ util.extend( corefn, {
     return this._private.container;
   },
 
+  mount: function( container, rendererOptions ){
+    if( container == null ){ return; }
+
+    let cy = this;
+    let _p = cy._private;
+    let options = _p.options;
+    
+    let rOpts = util.extend( {}, options.renderer, { name: 'canvas' }, rendererOptions );
```

Comment:
Should we override instead of extend?  If I specified renderer options it would only be to specify everything new from scratch.

Reply:
I am not sure.  If you specify everything new from scratch then your options should override everything right?  If someone wants to only specify a few new options this approach would also support that I think.

Reply:
Say I did

```js
let cy = new Cytoscape({ container: div1, renderer: { name: 'canvas', foo: 'bar', baz: 'bat' } }); // 1

cy.mount( div2, { name: 'webgl', baz: 'bop' } ); // 2
```

At 2, we have `{ name: 'webgl', baz: 'bop', foo: 'bar' }` but I didn't ask for `{ foo: 'bar' }`.

Reply:
The only usecase for setting the renderer options differently than init is if you want to change the rendering technology / renderer class used.  If you do that, you probably don't want old values around -- they could potentially conflict in the worst case.

Reply:
So get rid of ```options.renderer``` in the extend call?

Reply:
Makes sense.